logic/result.tsx

```diff
@@ -0,0 +1,44 @@
+import { Match, addPick } from "../utils/api";
+import { PickProps } from "../server/db";
+import { useEffect, useState } from "react";
+
+export function result() {
+  const [matches, setMatches] = useState<Match[]>(null);
+
+  const [picks, setPicks] = useState<PickProps[]>(null);
+
+  useEffect(() => {
+    fetch("/api/bundesligaMatches")
+      .then((response) => response.json())
+      .then(setMatches);
+    fetch("/api/picks")
+      .then((response) => response.json())
+      .then(setPicks);
+  }, []);
+  if (!matches) {
+    return <div> Loading...</div>;
+  }
+  if (!picks) {
```

Comment:
is it possible to use one statement? because the return is the same

Reply:
That's a good idea, I think. Something like:
```suggestion
  if (!matches || !picks) {
    return <div> Loading...</div>;
  }
```